easybuild/easyconfigs/d/DCA++/DCA++-CMake-CrayGNU-2015.11-cuda.eb

```diff
@@ -8,8 +8,8 @@ versionsuffix = '-cuda'
 #preinstallopts: change build_directory
 #disable clean build directory
 
-homepage = 'http://www.itp.phys.ethz.ch/people/schulthess'
-description = """This modules bundles all the dependencies for DCA++, the best DCA ever"""
+homepage = 'http://www.itp.phys.ethz.ch/research/comp/computation.html'
+description = """The DCA++ software project is a C++ implementation of the dynamical cluster approximation (DCA) and its DCA+ extension. It aims to solve lattice models of strongly correlated electron systems. This module bundles all the dependencies."""
```

Comment:
I would change the last sentence to "This module bundles all the dependencies for the CPU+GPU version."